src/Object/Api/Mastodon/Status.php

```diff
@@ -109,7 +109,7 @@ public function __construct(array $item, Account $account, Counts $counts, UserA
 		}
 
 		$this->sensitive    = $sensitive;
-		$this->spoiler_text = $item['title'] ?: $item['content-warning'];
+		$this->spoiler_text = ($item['title'] ?: $item['content-warning']) ?: '';
```

Comment:
Coalescing operators aren't ambiguous the way ternary operators are. Here, variables are evaluated from left to right.
```suggestion
		$this->spoiler_text = $item['title'] ?: $item['content-warning'] ?: '';
```